clients/src/main/java/org/apache/kafka/clients/admin/RaftVoterEndpoint.java

```diff
@@ -91,10 +91,6 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "RaftVoterEndpoint" +
-            "(name=" + name +
-            ", host=" + host +
-            ", port=" + port +
-            ")";
+        return name + "://" + host + ":" + port;
```

Comment:
can we do something like this?
```
if (name.contains(":")) {
    return name + "://[" + host + "]:" + port;
} else {
    return name + "://" + host + ":" + port;
}
```
to get around the ipv6 brain damage?

Reply:
^ sure thing 
(taking it you meant if host.contains(":"))